src/main/java/seedu/mtracker/commons/Validate.java

```diff
@@ -43,7 +43,7 @@
 
     public static boolean isInvalidNameCondition(String name, String instrumentType) {
         if (instrumentType.equals(AddForexParser.INSTRUMENT_TYPE)) {
-            return (name.length() != FX_PAIR_NAME_LENGTH);
+            return (name.length() != FX_PAIR_NAME_LENGTH | !name.matches("[a-zA-Z]+"));
```

Comment:
can I refer to https://github.com/AY2122S1-CS2113T-T12-1/tp/issues/173#issuecomment-956110579

I know we discussed this a few weeks ago, but it seems like this restriction even brought up the issue in the first place. I need to know your opinion on why forex specifically must have these special restrictions

Reply:
Yea, forex pairs are always 6 letters like USDJPY, AUDUSD, etc. and there are no numbers. !name.matches("[a-zA-Z]+") makes sure that there are no numbers input in place of 6 letters.

I think I would include these examples in the error message for clarity then. :)

Reply:
It can be like this USD/SGD, like most of the time they display it like this

Reply:
Ok then should we change the max length to 7 then. So the user can input either USD/SGD or USDSGD. But cannot  be more than 7 characters and also no numbers allowed in name?

Reply:
Whats the limit to restriction? Say we restrict numbers what abt special characters?

Reply:
Hi so this will be the new regex condition
<img width="684" alt="Screenshot 2021-11-01 at 10 06 57 PM" src="https://user-images.githubusercontent.com/55841532/139684683-69d5af55-7e90-4002-90b9-caae26acc046.png">
:

This makes sure that forex name contains either just 6 letters or 6 letter with a slash in the middle. All other non-alphabetic characters will be ignored.